packages/dev/core/src/Misc/screenshotTools.ts

```diff
@@ -40,7 +41,8 @@ export function CreateScreenshot(
     successCallback?: (data: string) => void,
     mimeType = "image/png",
     forceDownload = false,
-    quality?: number
+    quality?: number,
+    useFill = false

```

Comment:
Would it make sense to introduce an overload that takes an options object?

Reply:
Makes sense to me. But maybe in a separate PR?